internal/host/hostcommands/download_boot_artifacts_cmd_test.go

```diff
@@ -51,13 +51,20 @@ var _ = Describe("downloadBootArtifactsCmd.GetSteps", func() {
 		Expect(db.Create(&host).Error).To(BeNil())
 	})
 
+	AfterEach(func() {
+		// cleanup
+		ctrl.Finish()
+		common.DeleteTestDB(db, dbName)
+		stepReply = nil
+		stepErr = nil
```

Comment:
If these are not used in the setup/teardown code they probably should be scoped to the individual test.
That might be a better change here.

Reply:
Done.